instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/BannerAndVoiceInstructionsTest.kt

```diff
@@ -35,6 +36,7 @@ class BannerAndVoiceInstructionsTest : BaseTest<EmptyTestActivity>(EmptyTestActi
     }
 
     @Test
+    @Ignore("ignored due to NN-274")
```

Comment:
This instrumentation test now catches the issue from NN-274 @dkositsyn. I don't know why it didn't catch the issue on v121 but it does on v122. Here's a history file for reference - 
[2022-11-30T12-11-37Z_813afd44-3f43-460f-b3f1-ce0919380227.pbf.gz](https://github.com/mapbox/mapbox-navigation-android/files/10123385/2022-11-30T12-11-37Z_813afd44-3f43-460f-b3f1-ce0919380227.pbf.gz)

Reply:
Thanks, I'm working on the fix now, the reason is known already.